ui/tests/acceptance/secrets/backend/aws/aws-configuration-test.js

```diff
@@ -196,37 +194,12 @@ module('Acceptance | aws | configuration', function (hooks) {
       await runCmd(`delete sys/mounts/${path}`);
     });
 
-    test('it does not save lease AWS configuration if root configuration errored on save', async function (assert) {
```

Comment:
Is this the behavior that changed that you mentioned?

Reply:
correct!